src/main/java/bio/terra/service/dataset/flight/ingest/IngestMapKeys.java

```diff
@@ -5,4 +5,10 @@ private IngestMapKeys() {}
 
   public static final String INGEST_STATISTICS = "ingestStatistics";
   public static final String STAGING_TABLE_NAME = "stagingTableName";
+  public static final String BULK_LOAD_FILE_MODELS = "bulkLoadFileModels";
```

Comment:
This is yet another problem with the flight code; there's no way to ensure that these names are unique across all the different MapKeys classes.

One possible solution would be to add a prefix to each key based on the class name, assuming that we don't have duplicate map keys classes in our code base.

```
public final class IngestMapKeys {
  private IngestMapKeys() {}

  public static final String PREFIX = "ingest-"

  public static final String INGEST_STATISTICS = PREFIX + "ingestStatistics";
  public static final String STAGING_TABLE_NAME = PREFIX + "stagingTableName";
  public static final String BULK_LOAD_FILE_MODELS = PREFIX + "bulkLoadFileModels";
```

Reply:
Good idea.